ci/launch.sh

```diff
@@ -17,9 +15,7 @@ else
     ci/build-test.sh
 fi
 
-setAssemblyName "${PREFIX}"
-
-spatial cloud upload "${ASSEMBLY_NAME}" --log_level=debug --force --enable_pre_upload_check=false
+uploadAssembly "${ASSEMBLY_PREFIX}" "${PROJECT_NAME}"
```

Comment:
will that work? I don't see you specifying either of the two variables anywhere

Reply:
these are both defined in the buildkite step that calls this script